connect/res/values/strings.xml

```diff
@@ -5,4 +5,6 @@
     <string name="com_telenor_connect_error_text2">Please check your network settings or try again later.</string>
     <string name="com_telenor_connect_try_again">TRY AGAIN</string>
     <string name="com_telenor_connect_network_settings"><u>Network settings</u></string>
+    <string name="com_telenor_mobile_connect_login_button_text">Sign in with MobileConnect</string>
```

Comment:
In a way, it is, since I wanted them to be reviewed prior to having them translated. I'll request translations soon.